docs/tutorials/text_prediction/customization.md

```diff
@@ -1,274 +1,204 @@
-print('Label:', train_data['label'][row_index])
+from autogluon.text import ag_text_presets
+ag_text_presets.list_keys()
 ```
 
-## Perform HPO over a Customized Search Space with Random Search
+These pre-configured models use different backbones such as ELECTRA, RoBERTa, Multilingual BERT, and different fusion strategies. For example, `electra_small_fuse_late` means to use the ELECTRA-small model as the text backbone and use the late fusion strategy described in ":label:`sec_textprediction_architecture`". By default, we are using `default`, which is the same as `electra_base_fuse_late`. Next, let's try to specify the `presets` in `.fit()` to be `electra_small_fuse_late` and train a model on SSTs.
 
-To control which hyperparameter values are considered during `fit()`, we specify the `hyperparameters` argument.
-Rather than specifying a particular fixed value for a hyperparameter, we can specify a space of values to search over via `ag.space`.
-We can also specify which HPO algorithm to use for the search via `search_strategy` (a simple [random search](https://www.jmlr.org/papers/volume13/bergstra12a/bergstra12a.pdf) is specified below).
-In this example, we search for good values of the following hyperparameters:
-
-- warmup
-- learning rate
-- dropout before the first task-specific layer
-- layer-wise learning rate decay
-- number of task-specific layers
 
 ```{.python .input}
-import autogluon.core as ag
-from autogluon.text import TextPrediction as task
-
-hyperparameters = {
-    'models': {
-            'BertForTextPredictionBasic': {
-                'search_space': {
-                    'model.network.agg_net.mid_units': ag.space.Int(32, 128),
-                    'model.network.agg_net.data_dropout': ag.space.Categorical(False, True),
-                    'optimization.num_train_epochs': 4,
-                    'optimization.warmup_portion': ag.space.Real(0.1, 0.2),
-                    'optimization.layerwise_lr_decay': ag.space.Real(0.8, 1.0),
-                    'optimization.lr': ag.space.Real(1E-5, 1E-4)
-                }
-            },
-    },
-    'hpo_params': {
-        'search_strategy': 'random'  # perform HPO via simple random search
-    }
-}
+from autogluon.text import TextPredictor
+predictor = TextPredictor(path='ag_text_sst_electra_small', eval_metric='acc', label='label')
+predictor.set_verbosity(0)
+predictor.fit(train_data, presets='electra_small_fuse_late', time_limit=60, seed=123)
 ```
 
-We can now call `fit()` with hyperparameter-tuning over our custom search space. 
-Below `num_trials` controls the maximal number of different hyperparameter configurations for which AutoGluon will train models (5 models are trained under different hyperparameter configurations in this case). To achieve good performance in your applications, you should use larger values of `num_trials`, which may identify superior hyperparameter values but will require longer runtimes.
 
 ```{.python .input}
```

Comment:
Add comment that if users really care about one of these metrics, they should specify it as `eval_metric` before training so that AutoGluon can optimize for it.